test: #56 Write tests for slider with input component

kriLEBED opened the pull request:
Test cases included:

- it should renders correctly
- it should call onChange when slider is moved
- it should update inputValue correctly when input value is empty
- it should not update prices when input is blurred and value in input has not changed
- it should update prices when input is blurred and input is greater than max value

kriLEBED committed:
Create SliderWithInput.spec.tsx

test #56 Write tests for slider with input component

kriLEBED committed:
Update SliderWithInput.spec.tsx

fix: resolve lint errors in SliderWithInput.spec.tsx

- Improve typings for debounce mock (remove `any`)
- Alphabetize JSX props
- Remove unnecessary type assertions (use jest-dom matchers)
- Apply Prettier-friendly formatting

kriLEBED committed:
Update SliderWithInput.spec.tsx

fix(tests): update SliderWithInput.spec.tsx

- Remove extra spaces to satisfy Prettier

KorolyovKN commented on src/components/slider-with-input/SliderWithInput.spec.tsx:
```diff
@@ -0,0 +1,117 @@
+
+  it('should call onChange when slider is moved', () => {
+    render(
+      <SliderWithInput
```
Is it possible to move render to beforeEach?

kriLEBED replied:
Done. Move render to beforeEach.

kriLEBED committed:
Delete unnecessary file

kriLEBED committed:
test: #56 add test for slider with input component

Add unit tests for  SliderWithInput with beforeEach setup

Test cases included:
- it should renders correctly
- it should call onChange when slider is moved
- it should update inputValue correctly when input value is empty
- it should not update prices when input is blurred and value in input has not changed
- it should update prices when input is blurred and input is greater than max value

kriLEBED committed:
move render to beforeEach

kriLEBED committed:
fix parsing error

kriLEBED committed:
fix

kriLEBED committed:
fix prettier
